backends/tfhe-cuda-backend/cuda/src/integer/multiplication.cuh

```diff
@@ -188,7 +188,7 @@ __host__ void scratch_cuda_integer_sum_ciphertexts_vec_kb(
     int_radix_params params, bool allocate_gpu_memory) {
 
   size_t sm_size = (params.big_lwe_dimension + 1) * sizeof(Torus);
-  if (sm_size < cuda_get_max_shared_memory(gpu_indexes[0])) {
+  if (sm_size < cuda_get_max_shared_memory(0)) {
```

Comment:
Why not `gpu_indexes[0]`?

Reply:
I don't want to trigger conflicting configurations of the kernels on machines with several GPUs if they don't all have the same max SM size.